src-ui/lib/types/book.ts

```diff
@@ -1,4 +1,5 @@
 export type Book = {
+  id: number
```

Comment:
Let's make `id` a string. We'll generally want UUIDs. Unless you're intending to display it; in which case we can leave it as a human-readable format. Likely still string based though.